cypress/support/fragments/inventory/inventorySearchAndFilter.js

```diff
@@ -844,4 +848,16 @@ export default {
       Section({ id: 'acc02' }).find(KeyValue('Resource title')).has({ value: instanceData.title }),
     ]);
   },
+  clickNewOrder() {
+    cy.do([instanceDetailsSection.find(actionsButton).click(), newOrderButton.click()]);
+  },
+  varifyModalDialogExists() {
```

Comment:
No need this method
There is fragment/inventory/modals/newOrderModal
use NewOrderModal.waitLoading();